File: twiml/voice/pay/pay-7/pay-7.7.x.php
<?php
require_once './vendor/autoload.php';
use Twilio\TwiML\VoiceResponse;

$response = new VoiceResponse();
$pay = $response->pay();
$prompt = $pay->prompt(['for' => 'security-code', 'cardType' => 'amex']);
$prompt->say('Please enter security code for your American Express card. It’s the 4 digits located on the front of your card');

echo $response;

File: twiml/voice/pay/pay-7/pay-7.7.x.rb
require 'twilio-ruby'

response = Twilio::TwiML::VoiceResponse.new
response.pay do |pay|
  pay.prompt(for: 'security-code', card_type: 'amex') do |prompt|
    prompt
      .say(message: 'Please enter security code for your American Express card. It’s the 4 digits located on the front of your card')
  end
end

puts response
